README.md

```diff
@@ -117,8 +117,9 @@ Verify the successful installation by executing the provided command; confirmati
 2. The SQL file should encompass scripts intended for migration to Databricks SQL.
 
 Below is the detailed explanation on the arguments required for Transpile.
+- `transpiler [Required]` - Path to the transpiler configuration file.
```

Comment:
nit: this should be "transpiler-config-path", as we expect not the name of the transpiler nor the executable

Reply:
Done